KaitaiStream.js

```diff
@@ -802,6 +790,18 @@ KaitaiStream.createStringFromArray = function(array) {
   return chunks.join("");
 };
 
+KaitaiStream.twoU4sToS8 = function(high, low) {
+  if ((high & 0x80000000) != 0) {
```

Comment:
Always use strict equality operator `===` (or `!==` negated) in JavaScript unless you have a very good reason to use _loose equality_ op with type juggling (`==`/`!=`).

```suggestion
  if ((high & 0x80000000) !== 0) {
```

The loose equality op `==` knows several ways how to surprise you: at random `"" == false`, `"" == 0`, `[] == false` (whereas [`![] === false`](https://github.com/denysdovhan/wtfjs#true-is-not-equal--but-not-equal--too) actually) , `[] == ''`, `[] == 0`, `[0] == false`, `[1] == true`, `[null, null] == ","` etc. Thus it's _rarely_ a good idea to use the loose equality.

It doesn't matter whether you think you know the type of the operands, because it's JavaScript. Unless you are doing `typeof X === ...` everywhere, you don't know the types.